src/fields/fe_value_handler.cc

```diff
@@ -124,19 +120,18 @@ template <int elemdim, int spacedim, class Value>
 void FEValueHandler<elemdim, spacedim, Value>::value_list(const std::vector< Point >  &point_list, const ElementAccessor<spacedim> &elm,
                    std::vector<typename Value::return_type> &value_list)
 {
-    ASSERT_PTR(map_).error();
     ASSERT_EQ( point_list.size(), value_list.size() ).error();
 
     const DHCellAccessor cell = dh_->cell_accessor_from_element( elm.idx() );
     if (boundary_dofs_) this->get_dof_indices( elm, dof_indices);
     else cell.get_loc_dof_indices( dof_indices );
 
-    arma::mat map_mat = map_->element_map(elm);
+    arma::mat map_mat = MappingP1<elemdim,spacedim>::element_map(elm);
```

Comment:
I am not sure if the second template argument should be spacedim or 3 (as in the original object map_, see l. 106).